scrapyelasticsearch/scrapyelasticsearch.py

```diff
@@ -88,6 +89,10 @@ def from_crawler(cls, crawler):
 
         cls.validate_settings(ext.settings)
         ext.es = cls.init_es_client(crawler.settings)
+
+        if 'ELASTICSEARCH_INDEX_MAPPING' in ext.settings:
```

Comment:
not an expert on this tool yet but seems a good place to do it (right after ES init)